docs/variables.md

```diff
@@ -123,6 +123,7 @@ For `Template`-level metrics:
 | `workflow.creationTimestamp.<STRFTIMECHAR>` | Creation timestamp formatted with a [strftime](http://strftime.org) format character |
 | `workflow.priority` | Workflow priority |
 | `workflow.duration` | Workflow duration estimate, may differ from actual duration by a couple of seconds |
+| `workflow.parameters.scheduledTime` | Scheduled runtime formatted in RFC 3339 (only available for CronWorkflows) |
```

Comment:
this collides with any parameter with the same name, how about just `workflow.scheduledTime`?

Reply:
yes maybe. That's why initially I gave more specific `cronScheduledTime` if you want, we have `cronwfScheduleTime` or your suggession

It has to be a parameter so it will work flawlessly with our existing parameter logic. Current `workflow.x` variables are calculated during runtime or there is a field in `spec`. If we need to have `workflow.scheduledTime`, we need to add new field in `spec`

Reply:
I don't think it's too unreasonable to reserve a parameter name much like programming languages reserve keywords. But I agree with Alex that ideally we'd use `workflow.scheduledTime` or even `cronWorkflow.scheduledTime`

Reply:
Although at this point introducing `workflow.parameters.scheduledTime` might break users who already use that parameter name

Reply:
I think `scheduledTime` is fine

Reply:
@alexec and @simster7 Agreed. Let me go with `workflow.scheduledTime`

Reply:
I actually think `cronWorkflow.scheduledTime` is better... any opposition to that?

Reply:
@simster7  I would suggest sticking with `workflow.scheduledTime`, so code doesn't need to check another string for global variable

Reply:
Hmm, thinking about it more I think `workflow.` is right... it's the Workflow that got scheduled at this particular time, not the CronWorkflow